pyccel/codegen/pipeline.py

```diff
@@ -296,6 +296,14 @@ def handle_error(stage):
             handle_error('code generation')
             raise PyccelCodegenError('Code generation failed')
 
+        if language == 'python':
```

Comment:
do we have a problem in case of a program ?

Reply:
Yes. The main problem being that the python printer has no `_print_Program` method. But the file is generated in the correct location so I don't think there's a problem here

Reply:
okay